<launch>
  <node pkg="nodelet" type="nodelet" name="nodelet_manager" args="manager" />

  <node pkg="nodelet" type="nodelet" name="nodelet1"
        args="load depth_image_proc/point_cloud_xyzrgb nodelet_manager">
        <param name="approximate_sync" value="true" />
        <remap from="rgb/camera_info" to="/realsense_wrist/color/camera_info" />
        <remap from="rgb/image_rect_color" to="/realsense_wrist/color/image_raw" />
        <remap from="depth_registered/image_rect" to="/aligned_depth_image" />
        <remap from="depth_registered/points" to="/camera/color_point" />
  </node>
</launch>
